nilearn/glm/contrasts.py

```diff
@@ -47,7 +48,10 @@ def expression_to_contrast_vector(expression, design_columns):
     return contrast_vector
 
 
-def compute_contrast(labels, regression_result, con_val, contrast_type=None):
+@rename_parameters(
+    replacement_params={"contrast_type": "stat_type"}, end_version="0.13.0"
+)
```

Comment:
This decorator should take care of the deprecation cycle.

Will add a test to make sure of it.